vidarr-cli/src/main/java/ca/on/oicr/gsi/vidarr/cli/TestValidatorScript.java

```diff
@@ -19,15 +19,27 @@ public final class TestValidatorScript extends TestValidator {
   private String outputMetrics;
 
   @Override
-  public Validator createValidator() {
+  public Validator createValidator(String outputDirectory, String id, boolean verboseMode) {
     try {
-      final var tempDir = Files.createTempDirectory("vidarr-test-script");
-      final var calculateScript = tempDir.resolve("calculate");
-      final var calculateDir = tempDir.resolve("output");
-      Files.createDirectories(calculateDir);
-      Files.copy(Path.of(metricsCalculate), calculateScript);
+      // If output directory provided, we use that
+      // else we create a new directory in the default temporary-file directory
+      // Note: That path is associated with the default FileSystem which is UNIX in our case
+      final var finalDir = (outputDirectory != null) ? Path.of(outputDirectory)
```

Comment:
Since the temporary directory has a random sequence of numbers appended, the user could run tests in sequence, perhaps tweaking parameters, and get each version of the result saved to a different temporary directory. With the output directory set, the files have a consistent path and thus overwrite themselves. Would it make sense to create a new directory within the output directory (perhaps named after the current unix epoch) so that results do not overwrite each other?